components/StandardDialog.qml

```diff
@@ -38,8 +38,11 @@ import "effects/" as MoneroEffects
 
 Rectangle {
     id: root
-    color: "transparent"
+    color: MoneroComponents.Style.blackTheme ? "black" : "white"
     visible: false
+    radius: 10
+    border.color: MoneroComponents.Style.blackTheme ? MoneroComponents.Style.moneroGrey : MoneroComponents.Style._w_appWindowBorderColor
```

Comment:
Could we use ProcessingSplash border color here as well?

Reply:
ProcessingSplash border color is:
`MoneroComponents.Style.blackTheme ? Qt.rgba(255, 255, 255, 0.25) : Qt.rgba(0, 0, 0, 0.25)`

It works. But it's the same as the last review, where you said
> Not sure we should use transparency here (there is `appWindowBorderColor`, `grey`, etc.)

Reply:
Gotcha